packages/data/json/building-types/single-dwelling/wt-single-dwelling-9.9.9.test.ts

```diff
@@ -1,8 +1,8 @@
 // 3rd party
 import { describe, it, expect } from "vitest";
 // local
-import { isWalkthroughItemTypeVariable } from "../../src/useWalkthroughData";
-import data from "./single-dwelling-9.9.9.json";
+import { isWalkthroughItemTypeVariable } from "../../../src/useWalkthroughData";
+import data from "./wt-single-dwelling-9.9.9.json";
```

Comment:
what is the wt- stand for?

Reply:
shorthand for walkthrough. I wanted to differentiate between JSON that represented a specific walkthrough vs other site JSON.